x-pack/plugins/apm/server/lib/errors/get_error_group_sample.ts

```diff
@@ -43,7 +49,7 @@ export function getErrorGroupSample({
               { term: { [ERROR_GROUP_ID]: groupId } },
               ...rangeQuery(start, end),
               ...environmentQuery(environment),
-              ...esFilter,
+              ...searchQuery(kuery),
```

Comment:
what about kqlToQueryDSL 😬 ?

Reply:
or just `kqlToQuery`, or `queryFromKql`, you get the idea.

Reply:
So far these functions are all `*thing*Query`: `rangeQuery`, `environmentQuery`, `serviceNodeNameQuery`, etc. How about `kqlQuery`?

Reply:
yeah, works for me 👍

Reply:
(assuming you won't go with `kqlQuery` instead of `kuery` like Shahzad suggested)

Reply:
I'm just leaving `kuery` as the name of the query string parameter, and `kqlQuery` as the name of the function that converts the KQL query text into a `QueryContainer[]`.